src/models/model.cpp

```diff
@@ -9,9 +9,9 @@
 #include "whisper.h"
 #include "kernels.h"
 #ifdef USE_DML
-//  Because dml_provider_factory includes windows headers that #define min and max, this next line will prevent this from happening
-#define NOMINMAX
+#include <wil/wrl.h>
```

Comment:
How did you get past the min and max macros?

Reply:
We do it in the cmake files instead. It should be cleaner.